### DISCUSSION TITLE
Failed to bind to address /0.0.0.0:11800

### DISCUSSION THREAD

**Discussion by MuXueMaster** [apache/skywalking#8428] [Q&A]
I add a web hook  address in the configuration of alarm-settings.yml  . but it is not work . the picture for example:

webhook address:
![image](https://user-images.githubusercontent.com/44723876/149447708-cc834e7a-70b0-43a1-acd2-dc112a249745.png)

the  skywalking-oap-server.log has many error:
![XABNO`{TX`QH0FI7COXA742](https://user-images.githubusercontent.com/44723876/149447758-182a7ba7-1ae7-47fc-a9bd-46319dd964a3.png)

Did I do somethingwrong ? Pleas help me.

**Comment by wu-sheng** [comment]
There is nothing we can do about IP binding. This is your local env setup issue. This IP is not able to bind.

This error has nothing related to your alarm setting.

**Reply by wu-sheng** [reply]
Do you really kill your previous OAP instance?

**Reply by MuXueMaster** [reply]
Can restarting the server solve this problem？
